Hackney.Shared.HousingSearch/Factories/DomainFactory.cs

```diff
@@ -74,5 +81,53 @@ public static TempAccommodationOfficer ToDomain(this QueryableTempAccommodationO
                 LastName = entity.LastName
             };
         }
+
+        public static Domain.Asset.Contract ToDomain(this QueryableAssetContract entity)
+        {
+            return new Domain.Asset.Contract
+            {
+                Id = entity.Id,
+                TargetId = entity.TargetId,
+                TargetType = entity.TargetType,
+                StartDate = entity.StartDate,
+                ApprovalDate = entity.ApprovalDate,
+                IsApproved = entity.IsApproved,
+                Charges = entity.Charges?.ToDomain(),
+                RelatedPeople = entity.RelatedPeople?.ToDomain(),
+            };
+        }
+
+        public static Charges ToDomain(this QueryableCharges entity)
+        {
+            return new Charges
+            {
+                Id = entity.Id,
+                Type = entity.Type,
+                SubType = entity.SubType,
+                Frequency = entity.Frequency,
+                Amount = entity.Amount
+            };
+        }
+
+        public static IEnumerable<Charges> ToDomain(this IEnumerable<QueryableCharges> charges)
+        {
+            return charges.Select(x => x.ToDomain()).ToList();
```

Comment:
`.ToList()` is perhaps not needed here unless you want to specifically return list. Might be worth updating the method signature in that case.

Reply:
removed